### ISSUE TITLE
EthersEvent not defined (typescript)

### ISSUE THREAD

**Issue by eheimburg** [alchemyplatform/alchemy-sdk-js#62]
I'm trying to convert some javascript that uses the Alchemy SDK into TypeScript. It seems the typescript files in the project are malformed, making the library unusable with TypeScript. When I try to import the module, it fails. (This is 1.0.8)

This is my entire test script, it's one line:

`import { initializeAlchemy } from "@alch/alchemy-sdk";`

This results in 
```
node_modules/@alch/alchemy-sdk/dist/src/api/alchemy-websocket-provider.d.ts:3:10 - error TS2305: Module '"../internal/internal-types"' has no exported member 'EthersEvent'.

3 import { EthersEvent } from '../internal/internal-types';
           ~~~~~~~~~~~

node_modules/@alch/alchemy-sdk/dist/src/internal/internal-types.d.ts:10:12 - error TS2304: Cannot find name 'EthersEvent'.

10     event: EthersEvent;
              ~~~~~~~~~~~

node_modules/@alch/alchemy-sdk/dist/src/internal/internal-types.d.ts:45:64 - error TS2304: Cannot find name 'SubscriptionEvent'.

45 export declare type WebSocketMessage = SingleOrBatchResponse | SubscriptionEvent;
                                                                  ~~~~~~~~~~~~~~~~~


Found 3 errors.
```

If I look at the code, it really doesn't have a definition for `EthersEvent`. I'm not a TypeScript expert nor an Alchemy expert, but I don't see how this code could ever work and I'm not sure what to do to go about fixing it.

**Comment by thebrianchen** [comment]
@eheimburg Thanks for flagging this and filing an issue. I wasn't able to reproduce the issue in my dummy`ts-node` project, but I can see that the type is not exported. Do you mind uploading your setup to get this error? 

It's definitely a bug that we're not exporting `EthersEvent`, but I also want to catch any other types we're not currently exporting. Thanks!

**Comment by thebrianchen** [comment]
@eheimburg Thanks! I'm able to repro it. Turns out it's `tsc` that triggers the error, whereas `ts-node` is ok. Will investigate

**Comment by thebrianchen** [comment]
[Test.zip](https://github.com/alchemyplatform/alchemy-sdk-js/files/9144240/Test.zip)

@eheimburg Thanks for your repro! I created a new dist that I'll publish in a subsequent version that fixes the `EthersEvent` issue. The missing `node` and `AsyncIterable` imports can be fixed by running `npm install --D @types/node` in your root directory, which I've done. The zip file I've opened contains the modifications, and it's able to compile fine for me.
